### PR TITLE
Add v-chart reloadConfiguration method (CHARTS-211)

### PR DESCRIPTION
**Pull request by alvarezguille**
[Review on Reviewable](https://reviewable.io/reviews/vaadin/charts-component/37)

### CODE REVIEW

**Comment by ripla** [comment]
Maybe reword "Async not needed anymore"? E.g. " Remove async from v-series\n\nRemoved because..."

---

Reviewed 5 of 5 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions, all commit checks successful.

---

<sup>**[test/chart-reload-test.html, line 76 \[r1\]](https://reviewable.io:443/reviews/vaadin/charts-component/37#-JxiA1U05Q24Ob_Ggdz2)** ([raw file](https://github.com/vaadin/charts-component/blob/9551dadc328196489a5b014919b8b018443a97e0/test/chart-reload-test.html#L76)):</sup>
Could we do this with just the tag and remove the timeout? Which would be better?

---

<sup>**[test/chart-reload-test.html, line 84 \[r1\]](https://reviewable.io:443/reviews/vaadin/charts-component/37#-Jxi9noO0wgO-Wjh8jyR)** ([raw file](https://github.com/vaadin/charts-component/blob/9551dadc328196489a5b014919b8b018443a97e0/test/chart-reload-test.html#L84)):</sup>
test('test foobar... -> test('foobar...

---

<sup>**[v-chart.html, line 98 \[r1\]](https://reviewable.io:443/reviews/vaadin/charts-component/37#-JxiBz2S1Fl_yTOdjU3q)** ([raw file](https://github.com/vaadin/charts-component/blob/9551dadc328196489a5b014919b8b018443a97e0/v-chart.html#L98)):</sup>
Maybe put this into a separate function and fire the on-load event there as well.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/charts-component/37)

**Comment by alvarezguille** [comment]
Review status: all files reviewed at latest revision, 3 unresolved discussions, all commit checks successful.

---

<sup>**[test/chart-reload-test.html, line 76 \[r1\]](https://reviewable.io:443/reviews/vaadin/charts-component/37#-JxiA1U05Q24Ob_Ggdz2)** ([raw file](https://github.com/vaadin/charts-component/blob/9551dadc328196489a5b014919b8b018443a97e0/test/chart-reload-test.html#L76)):</sup>
Done.
I don't like having three identical charts in the test, that was easier with createlement.
In this case the tests can share the chart, but that doesn't happen always

---

<sup>**[test/chart-reload-test.html, line 84 \[r1\]](https://reviewable.io:443/reviews/vaadin/charts-component/37#-Jxi9noO0wgO-Wjh8jyR)** ([raw file](https://github.com/vaadin/charts-component/blob/9551dadc328196489a5b014919b8b018443a97e0/test/chart-reload-test.html#L84)):</sup>
Done.

---

<sup>**[v-chart.html, line 98 \[r1\]](https://reviewable.io:443/reviews/vaadin/charts-component/37#-JxiBz2S1Fl_yTOdjU3q)** ([raw file](https://github.com/vaadin/charts-component/blob/9551dadc328196489a5b014919b8b018443a97e0/v-chart.html#L98)):</sup>
Done.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/charts-component/37)

**Comment by ripla** [comment]
Reviewed 2 of 3 files at r2.
Review status: 4 of 5 files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/charts-component/37)

**Comment by ripla** [comment]
Reviewed 2 of 2 files at r3.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/charts-component/37)
